src/Server/modules/Analytics.js

```diff
@@ -201,5 +201,6 @@ module.exports = {
     GetTwoPlacesComparison: GetTwoPlacesComparison,
     GetRaceComparison: GetRaceComparison,
     GetUSPopulationAnalysis: GetUSPopulationAnalysis,
-    GetWorldPopulationAnalysis: GetWorldPopulationAnalysis
+    GetWorldPopulationAnalysis: GetWorldPopulationAnalysis,
+    GetMostRecentValue: GetMostRecentValue
```

Comment:
We will eventually delete this file, so maybe we want to copy GetMostRecentValue() function to the file you use this function.